Microsoft.Toolkit.Uwp.UI.Controls/SlidableListItem/SlidableListItem.xaml

```diff
@@ -51,6 +51,8 @@
     <Style TargetType="controls:SlidableListItem">
         <Setter Property="LeftForeground" Value="White" />
         <Setter Property="RightForeground" Value="White" />
+        <Setter Property="LeftBackground" Value="LightGray" />
```

Comment:
What are the current background values?

Reply:
What happens if the xaml setters are removed that the default (in code) is changed to null or transparent? Does the crash still happen?

Reply:
With null, no crash (but that would be a breaking change... and odd =P) - with Transparent there's a crash.

Reply:
I would prefer to see a fix-all solution. If setting to null works without having style setters, take that approach. This ensures that users that have applied a style will get the fix as well. We should then of course set the colors within the default style. 

Yes, this is a breaking style change, good news is that we're in a breaking change release

Reply:
Agree with @JohnnyWestlake, having default brushes is better than not having anything.